store.js

```diff
@@ -378,6 +378,15 @@ Store.prototype = {
         log.info("Adding repo " + repo.fullName);
         this.add(repo, cb);
     }
+,   updateRepo:    function (fullName, data, cb) {
+        this.getRepo(fullName, function(err, doc) {
+            if (err) return cb(err);
+            for (var k in data) doc[k] = data[k];
```

Comment:
I'm re-using the same pattern as the rest of the code; using `Object.assign` would probably be better now that we require node > 6, but I think that should be handled as a PR of its own.